R/zchunk_L124.nonco2_unmgd_R_S_T_Y.R

```diff
@@ -43,80 +41,186 @@ module_emissions_L124.nonco2_unmgd_R_S_T_Y_DISABLED <- function(command, ...) {
+              mutate(GFED_ForestFire_NOx, Non.CO2 = "NOx", type = "ForestFire"),
+              mutate(GFED_Deforest_NOx, Non.CO2 = "NOx", type = "Deforest")) %>%
+      # NMVOC is split into lots of inventories, so using CO for now.
+      bind_rows(mutate(GFED_ForestFire_CO, Non.CO2 = "NMVOC", type = "ForestFire"),
+                mutate(GFED_Deforest_CO, Non.CO2 = "NMVOC", type = "Deforest")) %>%
+      # Use CO for other missing gases. Previous code did this implicitly in lines 162-166
+      bind_rows(mutate(GFED_ForestFire_CO, Non.CO2 = "CH4", type = "ForestFire"),
+                mutate(GFED_Deforest_CO, Non.CO2 = "CH4", type = "Deforest"),
+                mutate(GFED_ForestFire_CO, Non.CO2 = "N2O", type = "ForestFire"),
+                mutate(GFED_Deforest_CO, Non.CO2 = "N2O", type = "Deforest")) %>%
+      gather(year, value, -Country, -Non.CO2, -type) %>%                                             # Convert from wide to long
+      mutate(year = as.integer(year)) %>%
+      spread(type, value) %>%                                                                        # Spread data so deforestation and forest fires are in columns
+      standardize_iso(col = "Country") %>%
+      change_iso_code('rou', 'rom') %>%                                                              # Convert Romania iso code to pre-2002 value
+      left_join(iso_GCAM_regID, by = "iso") %>%
+      # There are a set of iso codes in the GFED data that don't exist in the GCAM region mapping. Remove those now.
+      na.omit() %>%
+      group_by(GCAM_region_ID, Non.CO2, year ) %>%
+      summarize(ForestFire = sum(ForestFire), Deforest = sum(Deforest) ) %>%                         # Aggregate emissions by region, gas, and year
+      mutate(PctForestFire = ForestFire / (ForestFire + Deforest)) %>%                               # Compute share of emissions from forest fires
+      # There are regions where GFED data is zero for both forest fires and deforestation, leading to NAs
+      # Assume those missing values are places with 100% forest fires since these are easier to model in GCAM
+      replace_na(list(PctForestFire = 1)) %>%
+      mutate(PctDeforest = 1 - PctForestFire) %>%                                                    # Compute share of emissions from deforestation
+      select(-ForestFire, -Deforest) ->
+      FireShares_R_G_Y
+
+    # Downscale regional forest burning emissions to GLU based on the share of land in each GLU
+    # Use GFED to separate into forest fires and deforestation, which have different drivers in GCAM
+    L124.LC_bm2_R_UnMgdFor_Yh_GLU_adj %>%
+      group_by(GCAM_region_ID, year) %>%
+      mutate(land_share = value / sum(value)) %>%                                                      # Compute share of regional forest area in each GLU
+      select(-value) %>%
+      # There are places with land area but no emissions and vice versa. Use an inner_join to only get places with both
```

Comment:
Does this mean that mean iso's with forests but no emissions get zero EF? Could state that explicitly. (Perhaps add a TODO in here to consider applying an average EF?)